src/net/virtual_connection.rs

```diff
@@ -54,6 +59,28 @@ impl VirtualConnection {
         }
     }
 
+    /// Record that this connection has sent a packet. Returns whether the connection has
```

Comment:
```suggestion
    /// Records that this connection has sent a packet. Returns whether the connection has
```

Reply:
Plural start for function comment is a code style in lamianr.

Reply:
Good to know!